index.html

```diff
@@ -1,322 +1,287 @@
+                </span>
+                <span class="symbol">/</span>
+                <span>
+                <input placeholder="DD" id="Field12-2" name="Field12-2" type="text" class="field text day" value="" size="3" maxlength="2" tabindex="7">
+                </span>
+                <span class="symbol">/</span>
+                <span>
+                <input placeholder="YYYY" id="Field12" name="Field12" type="text" class="field text year" value="" size="4" maxlength="4" tabindex="8">
+                </span>
+                <span id="cal12">
+                </span>
+              </li>
+              <li class="buttons ">
+                <div>
+                  <input id="saveForm" name="saveForm" class="btTxt submit" type="submit" value="Submit">
+                </div>
+              </li>
+              <li class="hide">
+                <label for="comment">Do Not Fill This Out</label>
+                <textarea name="comment" id="comment" rows="1" cols="1"></textarea>
+                <input type="hidden" id="idstamp" name="idstamp" value="UmoBGA61EQfAYFBpO5r92qOFl6xKz1QUdyxgx7fsDog=">
+              </li>
+            </ul>
+          </form>
+        </li>
+      </ul>
+    </div>
+  </div>
+</div>
+<!-- End Contact Modal-->
+<!-- Footer -->
+<div id="footer-wrapper">
   <footer>
-    <p>
-      Copyright 2015 Yorkshire Interactive All Rights Reserved
-    </p>
-    <p class="disclaimer-text">
-      Disclaimer: no data is stored on our site or by Yorkshire Interactive. User analytics are
-      stored through Google Analytics and contact form submissions are stored by Wufoo.
+    <div class="">
+      <ul class="social">
+        <li><a href="https://www.facebook.com/yorkshireinteractive">Facebook</a></li>
+        <li><a href="https://twitter.com/yorkshireinc">Twitter</a></li>
+        <li><a href="https://github.com/yorkshireinteractive">Github</a></li>
+        <li><a href="">Designers and Developers</a></li>
+      </ul>
+    </div>
+    <p class="copyright">
+      &copy; 2016 Yorkshire Interactive. All Rights Reserved<br />
+      <a href="">Privacy Policy</a>
     </p>
   </footer>
-
-  <script src="assets/js/libs/jquery.min.js"></script>
-  <script src="assets/js/libs/jquery.fittext.js"></script>
-  <script src="assets/js/main.js"></script>
-  <script src="assets/js/wufoo.js"></script>
-
-<!--
-                    .-.
-                   ()I()
-              "==.__:-:__.=="
-             "==.__/~|~\__.=="
-             "==._(  Y  )_.=="
-  .-'~~""~=--...,__\/|\/__,...--=~""~~'-.
- (               ..=\=/=..               )
-  `'-.        ,.-"`;/=\ ;"-.,_        .-'`
-      `~"-=-~` .-~` |=| `~-. `~-=-"~`
-           .-~`    /|=|\    `~-.
-        .~`       / |=| \       `~.
-jgs .-~`        .'  |=|  `.        `~-.
-  (`     _,.-="`    |=|    `"=-.,_     `)
-   `~"~"`           |=|           `"~"~`
-                    |=|
-                    |=|
-                    |=|
-                    /=\
--->
```

Comment:
@dadambickford and I put this in here as a failsafe. Removing this in master will delete the website, domain name, and send an email out to every client.